tests/admin_scripts/tests.py

```diff
@@ -600,6 +600,21 @@ def test_setup_environ(self):
                 content,
             )
 
+    def test_setup_environ_app_config(self):
+        "directory: startapp creates the correct app config"
+        args = ['startapp', 'settings_test']
+        app_path = os.path.join(test_dir, 'settings_test')
+        out, err = self.run_django_admin(args, 'test_project.settings')
+        with open(os.path.join(app_path, 'apps.py'), 'r') as f:
```

Comment:
I think I would simply add these assertions to the above test to avoid repeating the setup process unless I missed a reason why that won't work.